docs/doxygen/pages.dox

```diff
@@ -145,8 +145,8 @@ Some configuration settings are C pre-processor constants, and some are function
 @section CELLULAR_CONFIG_STATIC_ALLOCATION_CONTEXT
 @copydoc CELLULAR_CONFIG_STATIC_ALLOCATION_CONTEXT
 
-@section CELLULAR_CONFIG_STATIC_ALLOCATION_COMM_CONTEXT
```

Comment:
Comm interface context is allocated in developer's porting now. Suggest to remove this config or we create another PR to review all the configs.

Reply:
Currently this change is for misra C, I agree to create another PR for the purpose of removing.